xclim/indicators/atmos/_conversion.py

```diff
@@ -215,3 +217,16 @@ class Converter(Indicator):
     ),
     compute=wrapped_partial(indices.wind_chill_index, mask_invalid=True),
 )
+
+
+potential_evapotranspiration = Converter(
+    identifier="potential_evapotranspiration",
+    units="mm day-1",
+    standard_name="water_evapotranspiration_flux",
+    long_name="Potential evapotranspiration",
+    description=(
+        "The daily or monthly potential for water evaporation from soil and transpiration by plants if the water "
```

Comment:
```suggestion
        "Potential for water evaporation from soil and transpiration by plants if the water "
```
The frequency of the data will be self explanatory, no need for it in the description, especially if it is ambiguous like here. (Same comment for the translation).